### DISCUSSION TITLE
Github version

### DISCUSSION THREAD

**Discussion by solokot** [TrianguloY/OpenInWhatsapp_Translation#42] [Ideas]
Greetings.
I think publishing the app to GitHub is a good idea in itself, but it's even better to make this version free of Google's restrictions: i.e. in this case with a built-in call list reader instead of a separate addon.

**Comment by TrianguloY** [comment]
Definitely the way to go. The current logic is inside the add-on, but it's relatively short so I need to test and try to include it here too.

I also want to include other minor modifications, I really need to investigate how to create different variants (other than changing the base color) and somehow automate and simplify the differences.
